src/Details.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+
+class Details extends Component {
+    constructor() {
+      super();
+      this.state = { detailsOfShow: "Hello, this will be the details page for each Movie & TV show :)" }
```

Comment:
This line is a bit long. Could you split it please?
```
this.state = { 
    detailsOfShow: "Hello, this will be the details page for each Movie & TV show :)" 
}
```